src/pages/payment-methods/_expanded-list.js

```diff
@@ -16,6 +16,7 @@ const StyledButton = styled(Button)`
 const StyledChevron = styled.img`
     height: 16px;
     width: 16px;
+    margin-left: 40px;
```

Comment:
Nice job, however I think that the chevron will no longer be centered on smaller screen widths such as mobile devices as the column widths are slightly different there. Hence, a predetermined 40px margin might not be the best option to cover all cases. A suggestion would be to use `display: flex` for the parent component and to have its `justify-content` and `align-items` set to center.

Reply:
Thanks, I'll work on that